.travis.yml

```diff
@@ -1,11 +1,9 @@
 language: python
 
-python: 3.5
+python: 3.7
 env:
-    - TOXENV=py27
-    - TOXENV=py33
-    - TOXENV=py34
-    - TOXENV=py35
+    - TOXENV=py37
+    - TOXENV=py38
     - TOXENV=pep8
```

Comment:
Please add support for Python 3.6.

Also please see https://github.com/morepath/morepath_cerebral_todomvc/commit/c5b413728e05e92423c162534c32e6b39ee1cb0f on how to fix the currently broken CI.

Reply:
Done